services/apis/nondbgapUsersync/nonDbgapTasks.js

```diff
@@ -0,0 +1,139 @@
+const chai = require('chai');
+const queryString = require('query-string');
+const { Bash } = require('../../../utils/bash.js');
+const { Gen3Response, checkPod } = require('../../../utils/apiUtil.js');
+
+const bash = new Bash();
+
+const I = actor();
+
+const TARGET_ENVIRONMENT = `${process.env.NAMESPACE}.planx-pla.net`;
+
+const scope = 'openid profile email ga4gh_passport_v1';
+
+module.exports = {
+  async runUserSyncDBGap() {
+    console.log('### Running usersync job with DBGap ...');
+    console.log(`start time: ${Math.floor(Date.now() / 1000)}`);
+    bash.runJob('usersync', args = 'ADD_DBGAP true'); // eslint-disable-line no-undef
+    await checkPod(I, 'usersync', 'gen3job,job-name=usersync');
+    console.log(`end time: ${Math.floor(Date.now() / 1000)}`);
+  },
+
+  async runUserSync() {
+    console.log('### Running usersync job ...');
+    console.log(`start time: ${Math.floor(Date.now() / 1000)}`);
+    bash.runJob('usersync');
+    await checkPod(I, 'usersync', 'gen3job,job-name=usersync');
+    console.log(`end time: ${Math.floor(Date.now() / 1000)}`);
+  },
+
+  async getRasToken() {
+    console.log('### Getting auth code ...');
+    const authURL = '/user/oauth2/authorize?'
+              + 'response_type=code'
+              + `&client_id=${I.cache.RAS_clientID}`
+              + `&redirect_uri=https://${TARGET_ENVIRONMENT}/user`
+              + `&scope=${scope}`
+              + '&idp=ras';
+    I.amOnPage(authURL);
+    // filling the RAS login form
+    I.fillField('USER', process.env.RAS_TEST_USER_1_USERNAME);
+    I.fillField('PASSWORD', process.env.RAS_TEST_USER_1_PASSWORD);
+    I.waitForElement({ xpath: 'xpath: //button[contains(text(), \'Sign in\')]' });
+    I.click({ xpath: 'xpath: //button[contains(text(), \'Sign in\')]' });
+    I.saveScreenshot('AfterSignIn.png');
+    // checking if the authcode is present in the url after login
+    I.seeInCurrentURL('code');
+    // grabbing the authcode from the url and storing it in the variable
+    const authCodeURL = await I.grabCurrentUrl();
+    const url = new URL(authCodeURL);
+    const authCode = url.searchParams.get('code');
+    // check if the authCode isnt empty
+    expect(authCode).to.not.to.be.empty;
```

Comment:
use chai.expect or add this `const { expect } = chai;` at start?

Reply:
Yes used the wrong import 
https://github.com/uc-cdis/gen3-qa/pull/818/files#diff-2dac60cb7af5a9a9349ef245b6bfda9231fb4b8c22f160fa66b4deecc66a07d7R1